src/param_spec.rs

```diff
@@ -547,3 +605,230 @@ impl ParamSpec {
         }
     }
 }
+
+pub trait ParamSpecType:
+    StaticType + FromGlibPtrNone<*mut gobject_sys::GParamSpec> + 'static
+{
+}
+
+extern "C" {
+    pub static g_param_spec_types: *const glib_sys::GType;
+}
+
+macro_rules! define_param_spec {
+    ($typ:ident, $ffi_typ:path, $mod_name:ident, $typ_offset:expr) => {
```

Comment:
You could rename `$ffi_typ` into `$ffi_type` and `$typ_offset` into `$type_offset`.

Reply:
Indeed, just seems inconsistent with `$typ` then :) Any suggestions for making it the same?

Reply:
`$rust_type`?

Reply:
Thanks, better :)